internal/telemetry/flags.go

```diff
@@ -0,0 +1,57 @@
+package telemetry
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Mode is the Telemetry Mode
+type Mode string
+
+// String returns the string representation
+func (m Mode) String() string {
+	val := string(m)
+	return val
+}
+
+// Type returns the Mode type
+func (m Mode) Type() string { return "string" }
+
+// Set validates and sets the output format value
+func (m *Mode) Set(val string) error {
+	mode := Mode(val)
+
+	if !isValidMode(mode) {
+		allModes := []string{OnSelected.String(),
```

Comment:
if youre going for the multi-line declaration here, then put each item on its own line (which means the closing `}` here would also be on its own line